spec/support/matchers.rb

```diff
@@ -48,6 +48,16 @@
       end
     end
 
+    doc.css("a[href^='https://secure.login.gov']").each do |a|
+      next if a[:lang]
+      page = a[:href]
+      link_path = URI::parse(page).path
+      next if locale == 'en'
+      if !link_path.start_with?("/#{locale}/") && link_path != "/#{locale}"
```

Comment:
what if we did something like
```suggestion
      if link_path.split('/').first != locale
```

Reply:
I slightly prefer the existing implementation, feels a bit clearer what we're checking for, but not strongly so.